books/free-programming-books-ja.md

```diff
@@ -276,6 +276,7 @@
 
 * [C++11の文法と機能(C++11: Syntax and Feature)](https://ezoeryou.github.io/cpp-book/C++11-Syntax-and-Feature.xhtml) - 江添亮
 * [C++入門](https://www.asahi-net.or.jp/~yf8k-kbys/newcpp0.html) - 小林健一郎
+* [C++入門 AtCoder Programming Guide for beginners (APG4b)](https://atcoder.jp/contests/APG4b) - 齋藤主裕, 石黒淳
```

Comment:
Are there a spacing typo in attributions?

```suggestion
* [C++入門 AtCoder Programming Guide for beginners (APG4b)](https://atcoder.jp/contests/APG4b) - 齋藤 主裕, 石黒 淳
```

Reply:
Thanks for the review!

Regarding the space.
When writing Japanese names in Kanji, the meaning remains the same with or without spaces.
I removed the space because I thought it would be better to unify the formatting.

Reply:
We should use the same text as provided by links. In this case there are an space :wink: